### DISCUSSION TITLE
Welcome to ZITADEL discussions!

### DISCUSSION THREAD

**Discussion by fforootd** [zitadel/zitadel#1195] [General]
Hi There 👋 

If you would like to engage (questions, ideas, use-case, ...) with the ZITADEL community feel free to use this board for all related discussions.

In case of a bug, please consider opening an [issue](https://github.com/caos/zitadel/issues/new/choose) and for security related information (disclosure) please follow our [security policy](https://github.com/caos/zitadel/security/policy).

For the ZITADEL documentation checkout out [docs.zitadel.ch](https://docs.zitadel.ch/)

Let us know what you think!

**Comment by fforootd** [comment]
## What will happen

We will ...

- Create and document a easy way of installing ZITADEL (including it's database) with a Kubernetes operator in your cluster 🚀 
- Publish a High Level Roadmap 🗺️

**Reply by fforootd** [reply]
The way to install ZITADEL in your own cluster can be found[ here ](https://docs.zitadel.ch/docs/guides/installation/crd)

**Comment by sreejith-geodis** [comment]
Please suggest if we can use the database as Postgres with Zitadel for production with an enterprise edition.

**Reply by mffap** [reply]
Please see our recommendation for production here: https://zitadel.com/docs/self-hosting/manage/production#database
Postgres is supported as part of an enterprise subscription: https://zitadel.com/docs/support/software-release-cycles-support#enterprise-supported

**Comment by sanjaypaudel35** [comment]
Hello zitadel team,

I have successfully setup the zitadel on my server, But when i try to update the user profile using api below using the user's access token
https://{MYZITADELDOMAIN.COM}/auth/v1/users/me/profile

It yields below error:
{
    "code": 16,
    "message": "Errors.Token.Invalid (AUTH-7fs1e)",
    "details": [
        {
            "@type": "type.googleapis.com/zitadel.v1.ErrorDetail",
            "id": "AUTH-7fs1e",
            "message": "Errors.Token.Invalid"
        }
    ]
}

But i am able to fetch the userinfo using the same access token.
https://{MYZITADELDOMAIN.COM}/oidc/v1/userinfo.

Could you please help me with this ASAP.

**Reply by komalshres** [reply]
I am facing similar issue. Urgent :-(

**Reply by hifabienne** [reply]
Have you read our guides on how to access the ZITADEL APIs? https://zitadel.com/docs/guides/integrate/access-zitadel-apis
I can imagine that you are missing the audience to access the APIs. You can request it with the following scope: urn:zitadel:iam:org:project:id:zitadel:aud

Side note: this is not the right place for your question, please open up a new discussion with your question or use our discord chat: zitadel.com/chat

**Reply by sanjaypaudel35** [reply]
I have provided above scope as it is but it still not working.

Ok i got it i, will open up new discussion. :)

**Comment by rolnikrolnik** [comment]
Hey, I am playing with Zitadel and was wondering where logs from Zitadel Actions end up? I mean those that use `logger.log` from `zitadel/log` modul that can be seen in multiple Actions' examples.

**Comment by pavithrainhlth** [comment]
Hey,  We have used zitadel in our application, but recently (9/3/2025) we noticed that Meta data is not showing up Zitadel UI. If anyone is facing the same issue kindly provide the solution if you find any

**Reply by fforootd** [reply]
This sounds like https://github.com/zitadel/zitadel/pull/9490 which should be fixed soon.

If you self-host you can downgrade again to make it work.

The metadata will still be there and you should be able to access it through the API.

**Comment by zitadelraccine** [comment]
Hey there!

For those that stop by this thread - Our main community hub is currently our Discord community [(click here to join)](https://discord.gg/Uc6VZtVftx)! You can still create issues here on GitHub for bugs, feature requests/proposals, or highlighting missing/incorrect docs but for discussions & peer-driven support, please join our community and open a new thread.
